modules/mauth-signer-akka-http/src/main/scala/com/mdsol/mauth/http/MAuthHeaders.scala

```diff
@@ -36,3 +36,35 @@ object `X-MWS-Time` extends ModeledCustomHeaderCompanion[`X-MWS-Time`] {
 
   override def parse(value: String): Try[`X-MWS-Time`] = Try(new `X-MWS-Time`(value))
 }
+
+final class `MCC-Authentication`(token: String) extends ModeledCustomHeader[`MCC-Authentication`] {
```

Comment:
No longer needed. Same for `MCC-Time`

Reply:
removed the both MCC-Authentication and MCC-Time